### PR TITLE
Add remove_worker method to CompanyResult

### PR DESCRIPTION
**Pull request by sloschert**
See #1092.

Plan: 8ec03a5b-3dbe-465d-a533-4b3bfe16121b

### CODE REVIEW

File: arbeitszeit_flask/database/repositories.py [hunk 1]
```diff
@@ -731,6 +731,15 @@ def add_worker(self, member: UUID) -> int:
...
         return companies_changed
 
+    def remove_worker(self, member: UUID) -> int:
+        companies_changed = 0
+        member = models.Member.query.filter(models.Member.id == str(member)).first()
+        assert member
+        for company in self.query:
+            companies_changed += 1
+            company.workers.remove(member)
+        return companies_changed
```
Comment (petsch0) [inline arbeitszeit_flask/database/repositories.py:741]: I know it is done the same way in add_worker but I still wonder if there is and/or will be in the future, a use case where self.query can contain more than one company? Isn't this action always triggered from exactly one company?

File: arbeitszeit_flask/database/repositories.py [hunk 2]
```diff
@@ -731,6 +731,15 @@ def add_worker(self, member: UUID) -> int:
             company.workers.append(member)
         return companies_changed
```
Comment (Amittai-Aviram) [inline arbeitszeit_flask/database/repositories.py:732]: Sorry to  hold things up, but do we expect ever to use this return value?  It looks as if we never do for add_worker — e.g., in arbeitszeit/use_cases/answer_company_work_invite.py, we have this:
`            self.database_gateway.get_companies().with_id(invite.company).add_worker(invite.member)`
It does not hurt, of course.

**Review by Amittai-Aviram — APPROVED** [review]
I have one question, but I am still approving.  The test looks adequate.

File: arbeitszeit_flask/database/repositories.py (hunk 2 above)
Comment (sloschert) [reply arbeitszeit_flask/database/repositories.py:732]: Yes, it seems to me that we do not use it currently. But as you said, it does not hurt..

**Review by petsch0 — APPROVED** [review]

**Comment by sloschert** [comment]
1xCR

**Pull request merged**